com.arteeh.Flasher.yml

```
@@ -0,0 +1,62 @@
+app-id: com.arteeh.Flasher
+# Libhandy depends on Vala, so the freedesktop platform is not enough
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: /app/bin/pinetime-flasher
```

```suggestion
command: pinetime-flasher
```

It's a nitpick, but technically `/app/bin` is already in the $PATH inside the sandbox so no need to use absolute path.